torch/ao/quantization/fx/prepare.py

```diff
@@ -244,18 +265,14 @@ def is_pattern_dtype_config_supported_by_backend(
     for dtype_config in dtype_configs:
         # check if arg dtype are supported
         supported = True
-        for arg in input_node.args:
-            supported = supported and \
-                is_input_arg_dtype_supported_by_backend(
-                    arg, input_node, node_name_to_target_dtype, dtype_config, backend_config)
-        for k, arg in input_node.kwargs.items():
-            supported = supported and \
-                is_input_arg_dtype_supported_by_backend(
-                    arg, input_node, node_name_to_target_dtype, dtype_config, backend_config)
+        for arg in list(input_node.args) + list(input_node.kwargs.values()):
+            supported = supported and is_input_arg_dtype_supported_by_backend(
```

Comment:
I think `is_input_arg_dtype_supported_by_backend` can check both activation and weight

Reply:
see: https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/fx/prepare.py#L191-L194